cmd/postgres-writer/main.go

```diff
@@ -114,40 +113,17 @@ func loadConfig() config {
 		SSLRootCert: mainflux.Env(envDBSSLRootCert, defDBSSLRootCert),
 	}
 
+	channelsCfgPath := mainflux.Env(envChannelsCfgPath, defChannelsCfgPath)
+	subtopicsCfgPath := mainflux.Env(envSubtopicsCfgPath, defSubtopicsCfgPath)
+	filters, _ := writers.LoadFiltersConfig(channelsCfgPath, subtopicsCfgPath)
```

Comment:
same here

Reply:
@manuio To fix this, I need your feedback about that: https://github.com/mainflux/mainflux/pull/1072#discussion_r391903974